app/models/user.rb

```diff
@@ -190,7 +190,8 @@ def will_save_change_to_email?
 
   def notifications_meta(account)
     {
-      unread_count: notifications.where(account_id: account.id, read_at: nil).count
+      unread_count: notifications.where(account_id: account.id, read_at: nil).count,
+      count: notifications.count
```

Comment:
@muhsin-k  should we scope the account when we are taking the count?

Reply:
Done